domains/hafi.json

```diff
@@ -0,0 +1,13 @@
+{
+	"owner": {
+		"username": "HafiTheCat",
+		"email": "",
+		"twitter": "",
+		"discord": "hafi_the_cat"
```

Comment:
Twitter section doesnt have a username, if you dont want to list a Twitter account then remove that line